configure.ac

```diff
@@ -5681,6 +5697,7 @@ echo "
 	Host:	       $host
 	Target:	       $target
 	GC:	       $gc_msg 
+	$thread_suspend_msg
```

Comment:
should we add this to the `gc_msg` instead? it already mentions whether sgen is concurrent, so e.g. something like this:

```
GC:            sgen (concurrent by default, hybrid suspend)
```

If we do that we should also fix up mono_gc_get_description() in sgen-mono.c to include this so it shows up for mono --version.

Reply:
Well the suspend mechanism is used by more than just SGen: it's also used by the debugger, by the managed suspend API, the thread abort machinery, etc.

Really pretty much the only thing that doesn't use it (AFAIK) is Boehm, when it's enabled, because it used its own version of preemptive suspend.

Reply:
Ok :)